docs/user/integrations/librenms.md

```diff
@@ -11,25 +11,47 @@ The LibreNMS SSoT integration is built as part of the [Nautobot Single Source of
 - Debug: Additional Logging
 - Librenms Server: External integration object pointing to the required LibreNMS instance.
 - hostname_field: Which LibreNMS field to use as the hostname in Nautobot. sysName or hostanme.
-- sync_location_parents: Whether to lookup City and State to add parent locations for geo locations.
+- location_type: This is used to filter which locations are synced to LibreNMS. This should be the Location Type that actually has devices assigned. For example, Site. Since LibreNMS does not support nested locations.
```

Comment:
```suggestion
- location_type: This is used to filter which locations are synced to LibreNMS. This should be the Location Type, such as Site, that actually has devices assigned since LibreNMS does not support nested locations.
```